website/pages/api-docs/auth/kubernetes/index.mdx

```diff
@@ -22,6 +22,11 @@ existence with the Kubernetes TokenReview API. This endpoint configures the
 public key used to validate the JWT signature and the necessary information to
 access the Kubernetes API.
 
+If they aren't specified, Vault will use the local CA cert and service account
+JWT for the `kubernetes_ca_cert` and `token_reviewer_jwt` parameters,
+respectively, when running in a Kubernetes pod. This behavior may be disabled by
+setting `disable_local_ca_jwt` to `true`.
```

Comment:
If users set this to true do they then also need to specify `kubernetes_ca_cert` and `token_reviewer_jwt` as well?

Reply:
If they set `disable_local_ca_jwt` to `true`, then either `kubernetes_ca_cert` or `pem_keys` needs to be specified as well. `token_reviewer_jwt` is optional.

Reply:
I'm not sure if it helped, but I tried to elaborate a little more in 7fe90f0.